app/full_text_search.py

```diff
@@ -20,19 +20,50 @@
 
 import model
 
-# The index name for full text search
-PERSON_FULL_TEXT_INDEX_NAME = 'person_information'
+# The index name for full text search.
+# This index contains person name and location.
+PERSON_LOCATION_FULL_TEXT_INDEX_NAME = 'person_location_information'
+
+def make_or_regexp(query_txt):
+    """
+    Craetes compiled regular expression for or search.
+    Args:
+        query_txt: Search query
+
+    Returns:
+        query_word | query_word | ...
+    """
+    query_words = query_txt.split(' ')
+    regexp = '|'.join([re.escape(word) for word in query_words if word])
+    return re.compile(regexp, re.I)
+
+def enclose_in_double_quotes(query_txt):
+    """
+    Enclosesd query_txt in double quotes.
```

Comment:
s/Enclosesd/Encloses/

Also you should say "Encloses each word in query_txt in double quotes." Otherwise it looks the whole query_txt is enclosed in a single quote.